### PR TITLE
Fix Failing Pipeline & Add Tests

### PR DESCRIPTION
**Pull request by notatiyyah**
- Fix styling error causing tests to fail
- Return an error object rather than throwing an error on the homepage (linting error)
- Tweak header/footer positioning css
- Add tests for the sidebar logic

### CODE REVIEW

File: src/pages/home-pages/home-pages.pages.jsx [hunk 1]
```diff
@@ -107,7 +107,7 @@ export const HomePage3 = () => {
         <a href = "https://playbook.hackney.gov.uk/api-specifications/ ">You can find our API Specification Playbook here</a>
         <br/><br/>
 
-    <h3>Why you might need an API</h3>
+    <h3>Why you might need an API ?</h3>
```
Comment (humulla) [inline src/pages/home-pages/home-pages.pages.jsx:110]:
```suggestion
    <h3>Why you might need an API?</h3>
```

File: cypress/integration/home.spec.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    expect(item.text()).to.eq(textArray[index]);
+  });
+}
+
+describe("Home page integration test", () => {
+  beforeEach(() => {
+    cy.visit("/")
+  });
+
+  it("Navbar test", () => {
```
Comment (notatiyyah) [inline cypress/integration/home.spec.js:30]: The header & footer aren't specific to the home page so the tests for them can probably go into another file :)

Comment (LBHLBasiu) [reply cypress/integration/home.spec.js:30]: I am planning to use this for Makers if that's ok

File: cypress/integration/home.spec.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    // find header items
+    const headerItems = header.get(".nav-items").children();
+    elementContainsTextArray(
+      headerItems,
+      headerItemText
+    );
+  });
+
+  it("Sidebar test", () => {
+    const sidebar = cy.get(".sidebar");
```
Comment (notatiyyah) [inline cypress/integration/home.spec.js:49]: Maybe test on multiple screen sizes (look at some of the other test files)

**Review by humulla — CHANGES_REQUESTED** [review]
There is integration tests for sidebar however can we add more for example:

- Check that the different links on the sidebar goes to the correct page

File: cypress/integration/home.spec.js (hunk 3 above)
Comment (LBHLBasiu) [reply cypress/integration/home.spec.js:49]: when i get some more time I will try to do it, however if someone has more time to test on multiple screen sizes is more than welcomed

Comment (notatiyyah) [reply cypress/integration/home.spec.js:49]: By that I mean implementing automated checks on various screen sizes. To do this, you can import the screen sizes like this:
```
import { screenSizes } from "../support/screenSizes";
```
then put the tests in a for loop like this:
```
screenSizes.forEach((screenSize) => {
   // tests here
}
```

See [this test file](https://github.com/LBHackney-IT/developers-hub-frontend/blob/master/cypress/integration/previewApi.spec.js) for an example

**Review by humulla — APPROVED** [review]

**Comment by notatiyyah** [comment]
> > There is integration tests for sidebar however can we add more for example:
> > 
> > * Check that the different links on the sidebar goes to the correct page
> 
> Screenshot 2021-11-02 at 14 32 39
> 
> is already checking that :D

I think @humulla means doing this by utilising the `click()` function. You're not clicking on the sidebar links and checking if they work as expected.

**Review by notatiyyah — COMMENTED** [review]
Also please remove the package-lock.json as we use yarn.

**Pull request merged**